python/ray/autoscaler/_private/commands.py

```diff
@@ -605,7 +604,7 @@ def kill_node(
 
     time.sleep(POLL_INTERVAL)
 
-    if config.get("provider", {}).get("use_internal_ips", False) is True:
```

Comment:
Nit: I'm not sure why it was written like this... Is it possible it's to guard against something like the string `"False"` being truthy in Python, or any other nonempty value or typo in the config? What do you think?

Reply:
Yeah I was also confused by that; it's inconsistent within that file which is why I had changed it when adding the additional check for the head node external IP param. For example, [this line](https://github.com/ray-project/ray/blob/7c1ef5e5be4d3427706abf7387aabafc95ec1873/python/ray/autoscaler/_private/commands.py#L1361). So either way, one of these lines should likely be changed. Let me know what you think.